libmcount/mcount.c

```diff
@@ -381,6 +384,7 @@ static void mcount_filter_init(enum uftrace_pattern_type ptype, char *dirname,
 	char *retval_str    = getenv("UFTRACE_RETVAL");
 	char *autoargs_str  = getenv("UFTRACE_AUTO_ARGS");
 	char *caller_str    = getenv("UFTRACE_CALLER");
+	char *loc_str       = getenv("UFTRACE_LOCATION");
```

Comment:
It also needs to set `needs_debug_info`.

Reply:
`needs_debug_info` is set when -L is applied in the record.c code below as I understand and tested.
Could you please let me know if I've missed anything?

```
		if (loc_str) {
			setenv("UFTRACE_LOCATION", loc_str, 1);
			setenv("UFTRACE_SRCLINE", "1", 1);
```

Reply:
Ah, I missed that.  So it's set by "UFTRACE_SRCLINE".  Thanks for the clarification.